geometry_msgs/Point point
float32 angleAccuracy
float32 distanceAccuracy
